hello-world/hello-world.js

```diff
@@ -51,15 +50,21 @@ app.onBeforeReplySent(request => {
   }
 });
 
-app.onState("likesVoxa?", request => {
-  if (request.intent.name === "YesIntent") {
-    return { tell: "doesLikeVoxa" };
-  }
+app.onState(
+  "likesVoxa?",
+  {
+    tell: "doesLikeVoxa"
+  },
+  "YesIntent"
+);
 
-  if (request.intent.name === "NoIntent") {
-    return { ask: "doesNotLikeVoxa" };
-  }
-});
+app.onState(
+  "likesVoxa?",
+  {
+    tell: "doesNotLikeVoxa"
+  },
+  "NoIntent"
```

Comment:
This was a feature intended since the beginning, finally got it working with code i actually feel good about